bin/servicerouter.py

```diff
@@ -96,6 +96,32 @@
   HAPROXY_BACKEND_HTTP_OPTIONS
     Sets HTTP headers, for example X-Forwarded-For and X-Forwarded-Proto.
 
+  HAPROXY_BACKEND_HTTP_HEALTHCHECK_OPTIONS
+    Sets HTTP health check options, for example timeout check and httpchk GET.
+    Parameters of the first health check for this service are exposed as:
+      * healthCheckPortIndex
+      * healthCheckProtocol
+      * healthCheckPath
+      * healthCheckTimeoutSeconds
+      * healthCheckIntervalSeconds
+      * healthCheckIgnoreHttp1xx
+      * healthCheckGracePeriodSeconds
+      * healthCheckMaxConsecutiveFailures
+      * healthCheckFalls is set to healthCheckMaxConsecutiveFailures + 1
```

Comment:
What does having this enable that you can't just do with healthCheckMaxConsecutiveFailures? The healthCheckFalls also isn't the most obvious name to me.